prowler/providers/aws/services/ec2/ec2_service.py

```diff
@@ -410,6 +410,18 @@ def __get_ebs_encryption_settings__(self, regional_client):
                 f"{regional_client.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
             )
 
+    def __enable_ebs_encryption_by_default__(self, region):
```

Comment:
Add a test for this in the service. Thanks!

Reply:
Done!